src/Compute/Compute/Manual/AddAzVmssRunCommand.cs

```diff
@@ -0,0 +1,253 @@
+
+
+                    foreach (VirtualMachineScaleSetVM vm in listOfVms)
+                    {
+                        var resultOfRunCmd = VirtualMachineScaleSetVMRunCommandsClient.BeginCreateOrUpdateAsync(resourceGroupName, vmScaleSetName, vm.InstanceId, this.Name, vmruncommand);
+                        var Pstemp = new PSVirtualMachineRunCommand();
+                        ComputeAutomationAutoMapperProfile.Mapper.Map<VirtualMachineRunCommand, PSVirtualMachineRunCommand>(resultOfRunCmd.Result, Pstemp);
+                        output.Add(Pstemp);
+                    }
+                    
+                    WriteObject(output);
+                }
+            });
+        }
+
+        [Parameter(
+            ParameterSetName = "DefaultParameter",
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true)]
+        [ResourceGroupCompleter]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            ParameterSetName = "DefaultParameter",
+            Position = 1,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true)]
+        [ResourceNameCompleter("Microsoft.Compute/virtualMachineScaleSets", "ResourceGroupName")]
+        public string VMScaleSetName { get; set; }
+
+        [Parameter(
+            ParameterSetName = "DefaultParameter",
+            Position = 2,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true)]
+        public string Location { get; set; }
+
+        [Parameter(
+            Mandatory = false)]
+        [AllowNull]
+        public string CommandId { get; set; }
+
+        [Parameter(
+            Mandatory = true)]
+        [AllowNull]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = false)]
+        [AllowNull]
+        public string ScriptPath { get; set; }
+
+        [Parameter(
+           Mandatory = false)]
+        [AllowNull]
+        public string ScriptUri { get; set; }
+
+        [Parameter(
+           Mandatory = false)]
+        [AllowNull]
+        public string Script { get; set; }
+
+        [Parameter(
+           Mandatory = false)]
+        [AllowNull]
+        public string TimeOutInSeconds { get; set; }
+
+        [Parameter(
+           Mandatory = false)]
+        [AllowNull]
+        public string ErrorBlobUri { get; set; }
+
+        [Parameter(
+            Mandatory = false)]
+        [AllowNull]
+        public string OutputBlobUri { get; set; }
+
+        [Parameter(
+           Mandatory = false)]
+        [AllowNull]
+        public string RunAsUser { get; set; }
+
+        [Parameter(
+            Mandatory = false)]
+        [AllowNull]
+        public SecureString RunAsPassword { get; set; }
+
+        [Parameter(
+            Mandatory = false)]
+        [AllowNull]
+        public Hashtable Parameters { get; set; }
```

Comment:
As in the design document, please use `Parameter` instead of `Parameters`

Reply:
fixed